<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialSettingsdialog</class>
 <widget class="QWidget" name="SerialSettingsdialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>258</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Serial Parameters</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="parityLabel">
          <property name="text">
           <string>Parity:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="baudRateComboBox"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="baudRateLabel">
          <property name="text">
           <string>Baud Rate:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QComboBox" name="stopBitsComboBox"/>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="parityComboBox"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="staopBitsLabel">
          <property name="text">
           <string>Stop Bits:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="dataBitsLabel">
          <property name="text">
           <string>Data Bits:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="dataBitsComb"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Modbus Parameters</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="1">
       <widget class="QSpinBox" name="responseTimeouSpinBox">
        <property name="minimum">
         <number>-1</number>
        </property>
        <property name="maximum">
         <number>5000</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="responseTimeoutLabel">
        <property name="text">
         <string>Resoins Timeout</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="numbOfEntrySpinBox"/>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="applayPushButton">
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="NumbOfEntryLabel">
        <property name="text">
         <string>Number of entries</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
